docs/examples.rst

```diff
@@ -130,6 +130,29 @@ Or you can use a dict::
     }
     new_issue = jira.create_issue(fields=issue_dict)
 
+You can even bulk create multiple issues::
+
+    issue_list = [
+    {
+        'project': {'id': 123},
+        'summary': 'First issue of many',
+        'description': 'Look into this one',
+        'issuetype': {'name': 'Bug'},
+    },
+    {
+        'project': {'key': 'FOO'},
+        'summary': 'Second issue',
+        'description': 'Another one',
+        'issuetype': {'name': 'Bug'},
+    },
+    {
+        'project': {'name': 'Bar'},
+        'summary': 'Last issue',
+        'description': 'Final issue of batch.',
+        'issuetype': {'name': 'Bug'},
+    }]
+    issues = jira.create_issues(field_list=issue_dict)
```

Comment:
This doesn't make sense. You've constructed `issues_list` but are not using it in this call.

Also, is it worth mentioning that the return value contains only the ones that were successfully created? (How do you programmatically find out which ones failed, e.g. if you want to retry them? Or mark them as failed somewhere upstream?)